src/irmin-pack/unix/file_manager.ml

```diff
@@ -176,28 +181,35 @@ struct
     let open Result_syntax in
     let* () = flush_suffix_and_its_deps t in
     let+ () = Index.flush ~with_fsync:t.use_fsync t.index in
+    (* NOTE index flush stats are updated in {!Pack_index} *)
     ()
 
   (* Auto flushes *********************************************************** *)
 
   (** Is expected to be called by the dict when its append buffer is full so
       that the file manager flushes. *)
-  let dict_requires_a_flush_exn t = flush_dict t |> Errs.raise_if_error
+  let dict_requires_a_flush_exn t =
+    Stats.Fm.incr_auto_dict ();
```

Comment:
I would suggest to only have `incr_dict_flushes` and not count the `auto` ones separately  (cc @Ngoguey42).

Reply:
No, let's could the `auto` ones separately. It corresponds to the number of time the dict buffer got full